sig-ops/README.md

```diff
@@ -0,0 +1,31 @@
+# SIG: Ops
+
+The Operations (Ops) Special Interest Group (SIG) provides a forum for groups that are operating Spinnaker deployments to coordinate their activities and identify and share best practices.
+* Production Readiness
+* Security Requirements
+* Monitoring and Logging
+* Performance and Scalability
+* HA & DR
+* Reference Architecture(s)  
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other TBD
+  * [Agenda](https://docs.google.com/document/d/1-YAEe1MrbgqtobyHyz2s5FvZFJGBM5wcYH-hE6sVgQw/edit?usp=sharing)
+
+## Leadership
+
+* Edgar Magana [emagana](https://github.com/emagana)
+* Cameron Fieber [cfieber](https://github.com/cfieber)
+* Chris Gruel [devorbitus](https://github.com/devorbitus)
+
+## Contact
+
+* [Slack](http://spinnakerteam.slack.com/messages/sig-ops)
+* [Mailing List](https://groups.google.com/a/spinnaker.io/forum/#!forum/sig-ops)
+* [Community Issues/PRs](https://github.com/spinnaker/spinnaker/labels/sig%2Fops)
+
+## Subprojects
+
+TBD
```

Comment:
nit: Are there expected to be sub-projects?  If not, I would just delete this section.

Reply:
Not right away. There will be space for specific areas such as Large Scale Deployment, Financial vs Enterprise, etc. I will get rid of it for now. Once we need them we will put it back if that is the case.